.github/workflows/security-scan.yml

```
@@ -11,9 +11,24 @@ env:
   ZAP_VERSION: 2.15.0
 
 jobs:
+  jobs:
```

actionlint (1.7.4)

"steps" section is missing in job "jobs"

(syntax-check)